packages/models/src/event.js

```diff
@@ -171,6 +182,14 @@ export default class Event {
     return null;
   }
 
+  get requestContentType() {
+    return Event.contentType(this.httpServerRequest, this.httpClientRequest);
+  }
+
+  get responseContentType() {
+    return Event.contentType(this.httpServerResponse, this.httpClientResponse);
+  }
```

Comment:
These are also `string | null | undefined` given the definition of `Event.contentType`